providers/aws-ec2/src/main/java/org/jclouds/aws/ec2/domain/VPC.java

```diff
@@ -40,17 +40,17 @@
       /**
        * The subnet is not yet available for use.
        */
-      PENDING, UNRECOGNIZED;
```

Comment:
Hm, hadn't meant to change VPC, just the classes I introduced - my thought was to throw an exception rather than accept an unknown value (what use would it be?).  However, I see that pattern is used quite extensively in the domain packages; so I'll just stick with that existing way of doing things.